src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,84 @@
+package lotto.domain;
+
+import static lotto.domain.LottoResult.REWARD_FIRST_GRADE;
+import static lotto.domain.LottoResult.REWARD_FOURTH_GRADE;
+import static lotto.domain.LottoResult.REWARD_NONE_GRADE;
+import static lotto.domain.LottoResult.REWARD_SECOND_GRADE;
+import static lotto.domain.LottoResult.REWARD_THIRD_GRADE;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LottoResult {
+
+    public static final int REWARD_FIRST_GRADE = 2000000000;
```

Comment:
```suggestion
    public static final int REWARD_FIRST_GRADE = 2_000_000_000;
```
하이픈을 활용하면 가독성을 좀 더 살릴 수 있습니다 ㅎㅎ